src/features/CourtCaseDetails/Tabs/Panels/TableRow.tsx

```diff
@@ -0,0 +1,15 @@
+import { Table } from "govuk-react"
+
+interface TableRowProps {
+  header: string
```

Comment:
since this is displayed side by side in a table row, maybe we could call this `label` or something else? `header` makes me think it's a label in a heading row at the top of the table

Reply:
This is a very good point, `label` is way better. I'll rename now.